server/adapters/dbtcloud.go

```diff
@@ -0,0 +1,167 @@
+package adapters
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/jitsucom/jitsu/server/jsonutils"
+	"github.com/jitsucom/jitsu/server/templates"
+	"io/ioutil"
+	"net/http"
+)
+
+const (
+	method        = "POST"
+	urlFormat     = "https://cloud.getdbt.com/api/v2/accounts/%d/jobs/%d/run/"
+	testUrlFormat = "https://cloud.getdbt.com/api/v2/accounts/%d/jobs/%d/"
+	bodyFormat    = `{"cause": "%s"}`
+)
+
+//DbtCloudConfig is a dto for parsing DbtCloud configuration
+type DbtCloudConfig struct {
+	AccountId int    `mapstructure:"accountId" json:"accountId,omitempty" yaml:"account_id,omitempty"`
+	JobId     int    `mapstructure:"jobId" json:"jobId,omitempty" yaml:"job_id,omitempty"`
+	Cause     string `mapstructure:"cause" json:"cause,omitempty" yaml:"cause,omitempty"`
+	Token     string `mapstructure:"token" json:"token,omitempty" yaml:"token,omitempty"`
+	Enabled   bool   `mapstructure:"enabled" json:"enabled,omitempty" yaml:"enabled,omitempty"`
+}
+
+//Validate returns err if invalid
+func (dcc *DbtCloudConfig) Validate() error {
+	if dcc == nil {
+		return errors.New("DbtCloud config is required")
+	}
+	switch {
+	case dcc.AccountId == 0:
+		return errors.New("'AccountId' is required parameter")
```

Comment:
It would be better to use the serialization field name here. Otherwise, users will see this error and might be confused about field names.